requirements.txt

```diff
@@ -1,5 +1,8 @@
 # required:
 requests<2.15,>=2.2
 pyxdg
+dnspython
 # optional:
 argcomplete
+future
+six
```

Comment:
Both `future` *and* `six`?  ;)

(I like `six`.  I don't like `future`, but I think I got it mixed up with `pies2overrides` in my mind, which is evil and actually caused me pain, when it got pulled in as a side effect of `isort` a while back).

Reply:
Anyway, these need to go into `install_requires` as well.

Reply:
OK, good remark. I used `six` only on one place and it was absolutely not necessary. So now it is reworked and in `install_requires`.